.travis.yml

```diff
@@ -220,4 +223,4 @@ deploy:
   on:
     repo: ${GITHUB_REPO_NAME}
     tags: true
-    condition: "$TRAVIS_TAG == *a*"
+    condition: "$TRAVIS_TAG != *[ab]*"
```

Comment:
Same comment as made elsewhere: `-l` should be added in front of `${ANACONDA_LABEL_ARG} and `condition` should be removed.

Reply:
I finally got it fixed... The sed line for ${ANACONDA_LABEL_ARG} needed a small tweak too.